experimenter/experimenter/kinto/tasks.py

```diff
@@ -188,6 +188,7 @@ def handle_updating_experiments(applications, records):
             experiment.publish_status = NimbusExperiment.PublishStatus.IDLE
             experiment.status_next = None
             experiment.published_dto = published_record
+            experiment.is_rollout_dirty = False
```

Comment:
I don't see any changes to the task tests to check for the cases where it goes from True > False?